cluster/addons/cluster-monitoring/googleinfluxdb/heapster-controller-combined.yaml

```diff
@@ -32,7 +32,8 @@ spec:
             - --sink=gcl
             - --sink=gcmautoscaling
```

Comment:
I'm not sure if GCM can handle a 1 minute frequency. Have you thought of GCM default quota limitations?